tajo-storage/tajo-storage-hdfs/src/main/proto/StorageFragmentProtos.proto

```diff
@@ -32,3 +32,14 @@ message FileFragmentProto {
   repeated string hosts = 5;
   repeated int32 disk_ids = 6;
 }
+
+message PartitionedFileFragmentProto {
+  required string id = 1;
+  required string path = 2;
+  required int64 start_offset = 3;
+  required int64 length = 4;
+  repeated string hosts = 5;
+  repeated int32 disk_ids = 6;
+  // Partition Name: country=KOREA/city=SEOUL
+  required string partitionName = 7;
```

Comment:
In my opionion, partitionKeys would be more proper because the attribute includes concatenated partition keys.